MAINTAINERS.md

```diff
@@ -0,0 +1,35 @@
+# Argoproj Maintainers
+
+[GOVERNANCE.md](https://github.com/argoproj/argoproj/blob/master/community/GOVERNANCE.md) describes the Argo governance.
+
+[membership.md](https://github.com/argoproj/argoproj/blob/master/community/membership.md) describes the responsibilities and requirements on the project roles. 
+
+## Maintainers
+
+| Maintainer | GitHub ID | Project Roles | Affiliation | Has Vote
+| --------------- | --------- | ----------- | ----------- | --------- | 
+| Saravanan Balasubramanian| [sarabala1979](https://github.com/sarabala1979) | Approver - Workflows | [Intuit](https://www.github.com/intuit/) | Yes |
+| Chetan Banavikalmutt | [chetan-rns](https://github.com/chetan-rns) | Approver - CD | [Red Hat](https://www.github.com/redhat/) | Yes |
+| Simon Behar| [simster7](https://github.com/simster7) | Approver - Workflows | [AirBnB](https://www.github.com/airbnb/) | Yes |
+| Xianlu Bird | [xianlubird](https://github.com/xianlubird) | Reviewer - Workflows | [Alibaba Cloud](https://github.com/aliyun) | Yes |
+| Henrik Blixt | [hblixt](https://github.com/hblixt) | Reviewer | [Intuit](https://www.github.com/intuit/) | Yes |
+| Shoubhik Bose | [sbose78](https://github.com/sbose78) | Reviewer | [Red Hat](https://www.github.com/redhat/) | Yes |
```

Comment:
Noticed there are a couple of reviewers without a project name as a suffix. What does that mean? Perhaps would be good to clarify this in https://github.com/argoproj/argoproj/blob/master/community/membership.md since it looks like the membership is scoped to subproject(s)?

Reply:
Good point, I'll make sure to update the membership.md as well, as the reviewer role has been expanded to also cover non-code contributors.